test/unit/org/apache/cassandra/index/sai/cql/LuceneAnalyzerTest.java

```diff
@@ -36,46 +34,37 @@ public void testQueryAnalyzer() throws Throwable
     {
         createTable("CREATE TABLE %s (id text PRIMARY KEY, val text)");
 
-        assertThatThrownBy(() -> createIndex("CREATE CUSTOM INDEX ON %s(val) USING 'StorageAttachedIndex' WITH OPTIONS = {" +
-                     "'index_analyzer': '{\n" +
-                     "\t\"tokenizer\":{\"name\":\"ngram\", \"args\":{\"minGramSize\":\"2\", \"maxGramSize\":\"3\"}}," +
-                     "\t\"filters\":[{\"name\":\"lowercase\"}]\n" +
-                     "}'," +
-                     "'query_analyzer': '{\n" +
-                     "\t\"tokenizer\":{\"name\":\"whitespace\"},\n" +
-                     "\t\"filters\":[{\"name\":\"porterstem\"}]\n" +
-                     "}'};"))
-        .hasCauseInstanceOf(ConfigurationException.class)
-        .hasRootCauseMessage("Properties specified [query_analyzer] are not understood by StorageAttachedIndex");
-    }
+        createIndex("CREATE CUSTOM INDEX ON %s(val) USING 'StorageAttachedIndex' WITH OPTIONS = {" +
+                    "'index_analyzer': '[\n" +
+                    "\t{\"tokenizer\":\"ngram\", \"minGramSize\":\"2\", \"maxGramSize\":\"3\"},\n" +
+                    "\t{\"filter\":\"lowercase\"}\n" +
+                    "]'," +
+                    "'query_analyzer': '[\n" +
+                    "\t{\"tokenizer\":\"whitespace\"},\n" +
+                    "\t{\"filter\":\"porterstem\"}\n" +
+                    "]'};");
```

Comment:
This doesn't look quite right. I think your rebase (or merge) based on the updated `vsearch` brought in old changes.

Reply:
Updated!